extensions/packageManager/src/node/index.ts

```diff
@@ -63,13 +63,14 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
       // if no sources are in the config file, set the default list to our 1st party feed.
       if (!packageSources) {
         packageSources = [
-          {
-            key: 'npm',
-            text: 'npm',
-            url: 'https://registry.npmjs.org/-/v1/search?text=keywords:bf-component&size=100&from=0',
-            searchUrl: 'https://registry.npmjs.org/-/v1/search?text={{keyword}}+keywords:bf-component&size=100&from=0',
-            readonly: true,
-          },
+          // TODO: Re-enable the NPM feed when we have a JS runtime
+          // {
+          //   key: 'npm',
+          //   text: 'npm',
+          //   url: 'https://registry.npmjs.org/-/v1/search?text=keywords:bf-component&size=100&from=0',
+          //   searchUrl: 'https://registry.npmjs.org/-/v1/search?text={{keyword}}+keywords:bf-component&size=100&from=0',
+          //   readonly: true,
+          // },
```

Comment:
On my machine, the npm feed is still present. We may need to actually remove it from the store for now.

Reply:
This will only happen to people who used the pre-release version.